app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -193,4 +180,32 @@ class ExoPlayerProfile(
 			subtitleProfile("idx", SubtitleDeliveryMethod.Embed)
 		)
 	}
+
+	companion object {
+		fun getDownmixSupportedAudioCodecs(): Array<String> =
```

Comment:
Make private and move to a value instead (`val downmixSupportedAudioCodecs = arrayOf(...)`)